### ISSUE TITLE
Propagate yaAGC and yaAGS changes from diff of dev-20070422 and dev-20090415 into SVN

### ISSUE THREAD

**Issue by GoogleCodeExporter** [samuelshark/virtualagc#14]
```
Update the SVN sources with the changes made by RSB and released on 
20090331. Since this source tree started with the 20070422 release the 
differences of these two releases shall be used to update yaAGC and yaAGS. 
Ron will take care of yaDSKY and the other GUI code with his new wxWidget 
implementation.

I made this issue of type defect because the new 2009 release has some bug 
fixes.


```

Original issue reported on code.google.com by `ohommes@gmail.com` on 7 Apr 2009 at 10:08

**Comment by GoogleCodeExporter** [comment]
```
Merged all changes from Ron's 20090415 yaAGC work. Still need to merge yaAGS as
promised but at least yaAGC is already done. The following should be noted:

- In the Makefile I have disabled for now the -static flag when USER is rburkey
because it forces me to have the static readline libs. 
- Also I force the use of READLINE since it was needed to get Code::Blocks to 
work
properly. Need to advise Ron of the significant changes I did in nbfget.c file
because there where issues with the pthread mutex'ing and pthread conditions
according to the readline manual so I fixed those and I intercepted a readline
function to fix and EOL issue for Code:Blocks debugging. Besides these minor 
notes
all changes are done for yaAGC.

I won't close this bug until yaAGS is also merged. Thereafter I'll discuss with 
Ron
how to get all the other VirtualAGC updates with regards to the new GUI and use 
of
wxWidgets.
```

Original comment by `ohommes@gmail.com` on 17 Apr 2009 at 4:27
- Changed title: **Propagate yaAGC and yaAGS changes from diff of dev-20070422 and dev-20090415 into SVN**
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Merge completed for both yaAGC as well as yaAGS. As noted before I disabled the
-static but will probably re-enable when I have the IMCROSS setup, configured 
and
working. Now we can look at the import of the other new elements of the 
VirtualAGC
which were added by Ron.
```

Original comment by `ohommes@gmail.com` on 18 Apr 2009 at 3:48
- Changed state: **Fixed**
